lib/game_collection.rb

```diff
@@ -0,0 +1,18 @@
+require 'csv'
+class GameCollection
+
+  @@game_path = './data/games.csv'
+
+  def initialize
+    rows = CSV.read(@@game_path, headers: true, header_converters: :symbol)
```

Comment:
Our discussion on @@ vs. @ was very helpful....and will be useful down the line!